src/influencetx/users/migrations/0003_auto_20200826_0033.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2020-08-26 00:33
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0002_auto_20200709_0256'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='user',
+            name='last_name',
+            field=models.CharField(blank=True, max_length=30, verbose_name='last name'),
+        ),
+    ]
```

Comment:
When I ran this it said it had more migrations to make, and `makemigrations` created a new one that changed `max_length=150`. Not sure why since it doesn't look to me like the user model was changed.